target-app/Home/Routes/HomeRoutes.swift

```diff
@@ -16,7 +16,9 @@ enum HomeRoutes: Route {
         case .home:
             let home = HomeViewController(
                 locationManager: LocationManager.shared,
-                viewModel: HomeViewModel(targetServices: TargetServices())
+                viewModel: HomeViewModel(
+                    targetServices: TargetServices()
+                )
```

Comment:
Are you getting any Swiftlint warning? if not pls leave it as it was before, since there is no other need to break lines

Reply:
done!